src/main/java/com/amazon/ion/impl/bin/IonRawBinaryWriter_1_1.kt

```diff
@@ -55,7 +55,7 @@ class IonRawBinaryWriter_1_1 internal constructor(
         /**
          * Clears this [ContainerInfo] of old data and initializes it with the given new data.
          */
-        fun reset(type: ContainerType? = null, position: Long = -1, isDelimited: Boolean = false) {
+        fun reset(type: ContainerType, position: Long, isDelimited: Boolean = false) {
```

Comment:
🗺️  Incidental change—I realize that the default values are always supplied for both `type` and `position`, so I removed the default values.